lib/pipeline/webpackProcess.js

```diff
@@ -38,26 +35,44 @@ class WebpackProcess extends ProcessWrapper {
     });
   }
 
-
-  webpackRun(filename, options, includeModules, importsList, detectErrors, callback) {
+  webpackRun(filename, options, includeModules, importsList, detectErrors, realCwd, callback) {
+    const self = this;
     let defaultOptions = {
+      mode: self.env === 'development' ? 'none' : 'production',
+      // devtool: self.env === 'development' ? 'source-map' : false,
+      // pipeline would need to copy .map files to dist/ target dir
+      // note: generating full source maps ('source-map') roughly doubles build time
       entry: fs.dappPath(filename),
       output: {
+        globalObject: 'typeof self !== \'undefined\' ? self : this',
```

Comment:
This is interpreted by webpack?
Also, you can just do `self ?`

Reply:
There's a bug in webpack 4, this is a workaround to get the expected UMD behavior.